sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamSuite.scala

```diff
@@ -976,24 +976,48 @@ class StreamSuite extends StreamTest {
       CheckAnswer(1 to 3: _*))
   }
 
-  test("streaming limit in complete mode") {
+  test("streaming limit before agg in complete mode") {
```

Comment:
nit: could you add the jira id to the test name? We recently added the following rule:

`Also, you should consider writing a JIRA ID in the tests when your pull request targets to fix a specific issue. In practice, usually it is added when a JIRA type is a bug or a PR adds a couple of tests to an existing test class. See the examples below:
Scala
test("SPARK-12345: a short description of the test") {
`

Reply:
aah right. thanks!